src/http.ts

```diff
@@ -566,7 +605,7 @@ export class LemmyHttp {
    * `HTTP.POST /comment`
    */
   async createComment(form: CreateComment) {
-    return this.wrapper<CreateComment, CommentResponse>(
+    return await this.wrapper<CreateComment, CommentResponse>(
```

Comment:
Revert all these awaits, they're pointless. The wrapper already returns a promise, so these functions just pass that promise onward.

Reply:
That's fair. Regarding pointless usages of keywords, do you think I should remove the `async`s from functions that look like this too? The `async` keyword allows you use `await` in a function, but these functions that call `this.wrapper` are going to return a `Promise` regardless of whether `async` is there or not. `async` is only really useful when your using `await`.

Reply:
I think they *might* have to remain async functions, because they're returning another async function, but ya you can try it.

Reply:
I removed the `async` keyword from most of the functions and they behave the same and have the same type as they did before. I left `async` on the functions that needed it (just `wrapper` and `uploadImage`).